test/k8sT/Policies.go

```diff
@@ -187,7 +187,7 @@ var _ = Describe("K8sPolicyTest", func() {
 		}
 
 		BeforeAll(func() {
-			namespaceForTest = helpers.GenerateNamespaceForTest()
+			namespaceForTest = helpers.GenerateNamespaceForTest("")
```

Comment:
Maybe the seed can be generated once at the beginning of all tests? Then it would definitely be unique between runs and callers don't need to remember which number to use in the seed if they change the tests.

Reply:
The seed is for making sure that if we have two namespaces (like in `Clusterwide policies` context), we are sure that generated names will be different.